lib/server/run.go

```diff
@@ -56,3 +65,57 @@ func Run(mux *http.ServeMux, grpcs *grpc.Server) {
 		log.Fatalf("Serve failed with error: %s", err)
 	}
 }
+
+// CloudRun starts an HTTP and gRPC server handling requests for all on the same
+// port, determined by the env var $PORT. If no HTTP mux or gRPC server is
+// provided (is nil), one with default routes/services will be started,
+// respectively.
+//
+// CloudRun() is a helper for servers that run in Google Cloud run, and thus
+// starts a server that supports the following protocols:
+//
+// ✔ grpc
+// ✗ grpc-web via websockets
+// ✗ HTTP 1.1
+// ✔ HTTP 2.0
```

Comment:
I'd like this matrix here (and in the other Run()) method to be covered by some form of unit test, so to catch regressions, and more easily allow us to iterate if necessary on a single main.

I'd add it as a TODO, though, this is a good starting point.

Reply:
Done